src/io.c

```diff
@@ -94,7 +96,7 @@ unsigned char io_event(unsigned char channel MARK_UNUSED)
 		break;
 
 	case SEPROXYHAL_TAG_TICKER_EVENT:
-		UX_TICKER_EVENT(G_io_seproxyhal_spi_buffer, {
+		CARDANO_UX_TICKER_EVENT(G_io_seproxyhal_spi_buffer, {
```

Comment:
I am not sure why do we need a macro for this. In fact, why don't we do
```
{ 
   UX_TICKER_EVENT(G_io_...., { /* normal sdk event handling */ })
   ... our callback handling
}
```

Reply:
I tried it and UX_ALLOWED seems to evaluate to `false` right after performing the `UX_TICKER_EVENT` macro, so even if our callback is called, it does not really execute ("No UX allowed, ignoring headless cb!" message is logged). Of course, when I "hacked" the `UX_ALLOWED` value and set it to true, the callback was executed and it seems to work ok, but I'm not sure about the side effects, since I guess, that the UX_ALLOWED is a guard against doing UX operations when you are not supposed to.